services/management/ia/templates_service.go

```diff
@@ -171,19 +290,38 @@ func (svc *TemplatesService) ListTemplates(ctx context.Context, req *iav1beta1.L
 			Severity:    managementpb.Severity(r.Severity),
 			Labels:      r.Labels,
 			Annotations: r.Annotations,
-			Source:      iav1beta1.TemplateSource_TEMPLATE_SOURCE_INVALID, // TODO
+			Source:      r.Source,
 		}
 
 		for _, p := range r.Params {
 			var tp *iav1beta1.TemplateParam
 			switch p.Type {
 			case alert.Float:
+				value, err := p.GetValueForFloat()
+				if err != nil {
+					return nil, errors.Wrap(err, "failed to get value for float parameter")
+				}
+
+				min, max, err := p.GetRangeForFloat()
+				if err != nil {
+					return nil, errors.Wrap(err, "failed to get range for float parameter")
+				}
+
 				tp = &iav1beta1.TemplateParam{
 					Name:    p.Name,
 					Summary: p.Summary,
-					Unit:    iav1beta1.ParamUnit_PARAM_UNIT_INVALID, // TODO
-					Type:    iav1beta1.ParamType_FLOAT,
-					Value:   nil, // TODO
+					Unit:    convertParamUnit(p.Unit),
+					Type:    convertParamType(p.Type),
+					Value: &iav1beta1.TemplateParam_Float{
+						Float: &iav1beta1.TemplateFloatParam{
+							HasDefault: true,           // TODO remove or fill with valid value.
+							Default:    float32(value), // TODO eliminate conversion.
```

Comment:
In API values for `Float` have type `float32`, but in `saas.Rule` it's `float64`